### DISCUSSION TITLE
Where did all the stars of this repo go? 😂

### DISCUSSION THREAD

**Discussion by amd64fox** [SpotX-Official/SpotX#21] [General]
I accidentally hid this repository not so long ago, because of this, all the stars were reset and the forks too 🤣
The github support said that they could not be restored.

Let's get them back quickly 👍

**Comment by vlad-solomon** [comment]
I gotchu bro

**Comment by amd64fox** [comment]
@vlad-solomon 
Thanks bro, I made your post a little prettier.

**Reply by vlad-solomon** [reply]
lmao thanks!

**Comment by amd64fox** [comment]
Restored in just a month, thank you all 🙏👏👏👏
